source/whereAny.js

```diff
@@ -0,0 +1,47 @@
+import _curry2 from './internal/_curry2';
+import _has from './internal/_has';
+
+
+/**
+ * Takes a spec object and a test object; returns true if the test satisfies
+ * the spec. Each of the spec's own properties must be a predicate function.
```

Comment:
I think everything in the first sentence after the semicolon needs to go. It's misleading to say that the test object has to satisfy the spec. Also, it looks like the docs for `where` have a typo in that same spot (test -> test object)

Reply:
@CrossEye what do you think here?

Reply:
- [  ] remove `'returns true if the test satisfies the spec.'`

Reply:
- [  ] remove 'returns true if the test satisfies the spec.'